src/styles/button.scss

```diff
@@ -60,6 +60,7 @@ $block: #{$fd-namespace}-button;
   line-height: 1rem;
   text-align: center;
```

Comment:
When pressing the button it gets the appropriate border color. And just after it gets the blue border, is it intended?

https://user-images.githubusercontent.com/20265336/140300705-12f23de8-e684-49a9-8110-0951c938ef1d.mov

Reply:
Yes, the blue border is the focus outline